README.md

```diff
@@ -1,252 +1,40 @@
 # datajudge
 
 [![CI](https://github.com/Quantco/datajudge/actions/workflows/ci.yaml/badge.svg)](https://github.com/Quantco/datajudge/actions/workflows/ci.yaml)
+[![Documentation Status](https://readthedocs.org/projects/datajudge/badge/?version=latest)](https://datajudge.readthedocs.io/en/latest/?badge=latest)
+[![Conda-forge](https://img.shields.io/conda/vn/conda-forge/datajudge?logoColor=white&logo=conda-forge)](https://anaconda.org/conda-forge/datajudge)
+[![PypiVersion](https://img.shields.io/pypi/v/datajudge.svg?logo=pypi&logoColor=white)](https://pypi.org/project/datajudge)
 
-Express and test specifications against data from database.
 
-# Why not `great expectations`?
+Express and test specifications against data from database.
 
-The major selling point is to be able to conveniently express expectations _between_ different `DataSource`s. `great expectations`, in contrast, focuses on expectations against single `DataSource`s.
 
-Over time, we have found some people find the following aspects of datajudge useful
-* Lots of the 'query writing' is off-loaded to the library by having `Constraint`s tailored to our needs
-* Easier/faster onboarding than with `great expectations`
-* Assertion messages with counterexamples and other context information, speeding up the data debugging process
+[Documentation](https://datajugde.readthedocs.io/en/latest/index.html)
 
 # Installation instructions
 
 `datajudge` can either be installed via pypi with `pip install datajudge` or via conda-forge with `conda install datajudge -c conda-forge`.
 
-You will likely want to use `datajudge` in conjuction with other packages - in particular `pytest` and database drivers relevant to your database. You might want to install `snowflake-sqlalchemy` when using snowflake, `pyscopg` when using postgres and platform-specific drivers ([Windows](https://docs.microsoft.com/en-us/sql/connect/odbc/windows/microsoft-odbc-driver-for-sql-server-on-windows?view=sql-server-ver15), [Linux](https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server?view=sql-server-ver15), [macOS](https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/install-microsoft-odbc-driver-sql-server-macos?view=sql-server-ver15)) when using mssql.
-
-For development _on_ `datajudge` - in contrast to merely using it - you can get started as follows:
-```bash
-git clone https://github.com/Quantco/datajudge
-cd datajudge
-mamba env create
-conda activate datajudge
-pip install --no-build-isolation --disable-pip-version-check -e .
-```
-
-# Example
-
-To get started, we will create a sample database using sqlite that contains a list of companies.
-
-```python
-import sqlalchemy as sa
-
-eng = sa.create_engine('sqlite:///example.db')
+# Usage
```

Comment:
I'd probably still keep here the explicit message for the database conenctors/drivers .

Reply:
[8cf792c](https://github.com/Quantco/datajudge/pull/17/commits/8cf792c78e3e277bfe6e484b7311cbfb1a49dfb7)